source/includes/gcp/_target_proxies.md

```diff
@@ -84,13 +89,62 @@ Retrieve a target proxy in an [environment](#administration-environments).
 | -------------------------------- | ------ |
 | `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
 | `id`<br/>*UUID*                  | The id of the backend service. |
-| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies.|
+| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies. |
```

Comment:
see comment above about HTTP/HTTPS only.

Reply:
ae81aa9d03315427864b7953c7932bce5c8f5420

Reply:
I still see this.

Reply:
3df058f313111b9807356745eb399f801937f674